src/CMakeLists.txt

```diff
@@ -187,6 +181,13 @@ set(Mapper_Common_SRCS
  gui/symbol/symbol_setting_dialog.cpp
  gui/symbol/symbol_properties_widget.cpp
  gui/symbol/symbol_point_editor.cpp
+
+ gui/map/map_widget.cpp
+ gui/map/map_editor.cpp
+ gui/map/map_editor_activity.cpp
+ gui/map/map_dialog_new.cpp
+ gui/map/map_dialog_scale.cpp
+ gui/map/map_dialog_rotate.cpp
```

Comment:
While we had to few folders in the past, we shouldn't fall into the other extreme now. There is not much to gain, but a lot more "../" to write (`#include "../../core/map_coord.h"`)